docs/question.md

```diff
@@ -62,9 +62,52 @@ Property | Type | Description
 `singleVariant` | boolean | Whether the question is not randomized and only generates a single variant. (Optional; default: `false`)
 `partialCredit` | boolean | Whether the question will give partial points for fractional scores. (Optional; default: `true`)
 `externalGradingOptions` | object | Options for externally graded questions. See the [external grading docs](externalGrading.md). (Optional; default: none)
+`dependencies` | object | External JavaScript or CSS dependencies to load.  See below.  (Optional; default: `{}`)
 
 For details see the [format specification for question `info.json`](https://github.com/PrairieLearn/PrairieLearn/blob/master/schemas/schemas/infoQuestion.json)
 
+### Question Dependencies
+
```

Comment:
I suppose a question would be does a similarly named element load that elements dependencies prior to loading the custom element? Or is it a clean break?

I'm 95% sure it's a clean break. But, this probably should be emphasized here to avoid CSS hackers thinking parts of an element can be re-used/extended.

Reply:
I'm not sure I understand this question?

Reply:
@mwest1066 

If there is an implementation of `pl-number-input`  in both the the **core elements/built in to PrairieLearn** and a **local course**, are these two implementations independent from each other? 

That is, I shouldn't expect the CSS from `pl-number-input` in the core to be available within the local course. Thus, if I wanted to change a single CSS rule it would require a complete duplication of all contents for `pl-number-input`.

Reply:
Right, my understanding is that only the course element will get loaded here.

Reply:
That's also my understanding, but I haven't checked the code.